packages/discord.js/src/structures/AttachmentBuilder.js

```diff
@@ -8,7 +8,7 @@ const { basename, flatten } = require('../util/Util');
 class AttachmentBuilder {
   /**
    * @param {BufferResolvable|Stream} attachment The file
-   * @param {APIAttachment} [data] Extra data
+   * @param {AttachmentData} [data] Extra data
```

Comment:
It seems this type definition doesn't exist. Mind creating it?

Reply:
It exists...https://github.com/discordjs/discord.js/blob/8683562704dcde1fa3d58dc6a1e774c267228f84/packages/discord.js/typings/index.d.ts#L3730

Reply:
It doesn't exist in JSDoc.

Reply:
oh, I assumed that the types in the TS typedef were automatically synced, I see now that's not the case. Fixed in ae86ea1607948d54b07d4ccdca80951e2be40a0c.